content/docs/user-guide/gems/gem-versioning.md

```diff
@@ -0,0 +1,90 @@
+title: Gem Versioning and Compatibility
+description: An overview of how Open 3D Engine Gems declare and use version, compatibility and dependency information.
+weight: 400
+---
+
+Gem versioning is an important way to convey the kind of changes made to a gem. 
+
+## O3DE Version Numbering
+
+O3DE uses `MAJOR.MINOR.PATCH` [semantic versioning](https://semver.org/) for all the `version` fields in `engine.json`, `gem.json` and `project.json` files.  
+
+- `MAJOR` is for API-breaking changes
+- `MINOR` is for non-API-breaking changes that add new APIs or change them in a non-breaking way
+- `PATCH` is for all other non-API-breaking changes, usually important fixes
+
+Example: If a breaking API change is made to a gem at version `2.0.1` the `MAJOR` version is incremented and the `MINOR` and `PATCH` are reset to `0` resulting in the new version being `3.0.0`.  See the [semantic versioning](https://semver.org/) page for more examples. 
+
+## O3DE Version Specifiers 
+
+Lists of compatible engines and dependencies use the `<name><version specifier>` format based on [PEP 440](https://peps.python.org/pep-0440/#version-specifiers).
+
+Examples:
+
+| Name | Version Specifier | Combined | Description |
+|------|-------------------|----------|-------------|
+| o3de | >=2.0.0           | o3de>=2.0.0 | o3de version 2.0.0 or greater |
+| o3de-sdk | == 1.2.0      | o3de-sdk==1.2.0 |  o3de-sdk version 1.2.0 | 
+| Atom | ~=2.0.0           | Atom~=2.0.0 | Atom version 2.0.0 up to but not including 3.0.0 |
+
+A gem that is known to be compatible with the an engine named `o3de-sdk` version `2.0.1` could include `o3de-sdk==2.0.1` in the `compatible_engines` field in `gem.json`
+
+
+## Gem Version And Compatibility Information In `gem.json` 
+
+Gem version, dependency and compatibility information is stored in the  `gem.json` file at the root of each gem. If the `gem.json` has no compatibility or dependency information in it, or the fields are empty, the gem is assumed to be compatible with all engines and have no gem dependencies. 
+
+
+| Field | Description |  Default |  Example | |
+|---------------------|---------------------|------------------------|------------------------|------------------------|
+| version | The `MAJOR.MINOR.PATCH` [semantic version](https://semver.org/) that is updated as changes are made to the gem. Developers may use the `dependencies` field in `gem.json` to indicate other gem dependencies and the version ranges they are compatible with.| 1.0.0 | 1.2.3 | |
+| compatible_engines | A list of engine names and version specifiers that this gem is known to be compatible with. If empty, the gem is assumed compatible with all engines if they meet all the requirements in the `engine_api_dependencies` and `dependencies` fields. | [ ] | [<br />  "o3de-sdk==1.2.0",<br />  "o3de>=2.0.0"<br />] | This gem is compatible with an engine named `o3de-sdk` version `1.2.0`, or any engine named `o3de` version `2.0.0` or greater. |
```

Comment:
I am guessing you are having an issue with adding a column to table that spans multiple lines?
![image](https://user-images.githubusercontent.com/56135373/235792971-fd149090-2129-4b17-8ed8-990e36816674.png)

Because ideally the version compatibility list would be in a code block for json.

HTML can actually be used to embed a table that supports multi line code blocks instead of using markdown tables: https://blog.eidinger.info/master-github-markdown-tables-with-multi-line-code-blocks


There is also example how to do it in the Archive Streaming Stack document markdown file that is in the sig-core repo: https://github.com/o3de/sig-core/blob/daa015fc865534dcb9fc4b0ba230ea362d923dd9/rfcs/rfc-core-2023-04-20-streaming-archive.md?plain=1#L238-L277

Reply:
nice, thanks I didn't know you could use html tables

Reply:
The table looks much better now!
![image](https://user-images.githubusercontent.com/56135373/235814530-728e8689-7045-42dc-ad24-6503dd5ffe8a.png)